js/custom.support.validate.js

```diff
@@ -154,9 +154,9 @@ function showSuccess() {
                         confirmButtonText: 'Ok'
                     });
                     resetForm();
-                } else if(false){  //condition to be used when response from server is returned
+                } else if(false){  //condition to be filled according to the returned response from server for suspicious user
```

Comment:
Do not put the words "Suspicious user" anywhere in our code. That is not appropriate. We already asked this below.

Also, you do not know why a non-successful response is given. There could be alot of reasons we don't get a response == 200.  So, just use a generic response. Generic comment.... and stop using the word "suspicious"

Reply:
Do you want to show same error in alert when recaptcha failed and when some error occurred at the server or you will show different messaage?

Reply:
Recaptcha doesn't "fail" so your question makes no sense.
Recaptcha just delivers a score.  If score isn't above a specific threshold, we will refuse to allow interaction with the back-end server to reduce spam load.
So, study recaptcha v3 deeper so you understand this "score" concept or reword your question so it aligns with the terminology found in Recaptcha v3.